### DISCUSSION TITLE
请问我的系统是win10,下载的版本是安装版，为‘lx-music-desktop-v2.7.0-x64-Setup’，但是下载好了，不能播放音乐是怎么回事呢？

### DISCUSSION THREAD

**Discussion by flight025** [lyswhut/lx-music-desktop#1913] [Q&A]
以这首新鸳鸯蝴蝶梦为例，点击播放，发现是灰白的，不能播放，怎么做？
![image](https://github.com/lyswhut/lx-music-desktop/assets/65944944/d211da21-83ab-4983-9dab-cc2188af52a0)

**Comment by limeng3150** [comment]
请问你是将洛雪音乐部署在自己服务器上面来供使用的吗

**Comment by flight025** [comment]
没有

**Comment by lovetuan** [comment]
没有导入源吧
